Source/Components/Row/MLBusinessRowView.swift

```diff
@@ -0,0 +1,371 @@
+            leftImageAccessoryImageView.bottomAnchor.constraint(equalTo: leftImageImageView.bottomAnchor),
+            leftImageAccessoryImageView.rightAnchor.constraint(equalTo: leftImageImageView.rightAnchor),
+        ])
+
+        NSLayoutConstraint.activate([
+            mainStackView.topAnchor.constraint(greaterThanOrEqualTo: containerView.topAnchor, constant: 16.0),
+            mainStackView.centerYAnchor.constraint(equalTo: containerView.centerYAnchor),
+            mainStackView.bottomAnchor.constraint(lessThanOrEqualTo: containerView.bottomAnchor, constant: -16.0),
+            mainStackView.leftAnchor.constraint(equalTo: leftImageImageView.rightAnchor, constant: 14),
+            mainStackView.rightAnchor.constraint(lessThanOrEqualTo: rightStackView.leftAnchor, constant: -14),
+        ])
+
+        NSLayoutConstraint.activate([
+            rightStackView.topAnchor.constraint(greaterThanOrEqualTo: containerView.topAnchor, constant: 18.0),
+            rightStackView.widthAnchor.constraint(greaterThanOrEqualToConstant: 80),
+            rightStackView.centerYAnchor.constraint(equalTo: containerView.centerYAnchor),
+            rightStackView.rightAnchor.constraint(equalTo: containerView.rightAnchor, constant: -16),
+        ])
+
+        NSLayoutConstraint.activate([
+            lineView.heightAnchor.constraint(equalToConstant: 1.0),
+            lineView.leftAnchor.constraint(equalTo: leftAnchor),
+            lineView.rightAnchor.constraint(equalTo: rightAnchor),
+            lineView.bottomAnchor.constraint(equalTo: bottomAnchor),
+        ])
+    }
+
+    public func update(with content: MLBusinessRowData) {
+        //CLEAN
+        rightStackView.arrangedSubviews.forEach { $0.removeFromSuperview() }
+        rightPrimarySecondaryStackView.arrangedSubviews.forEach { $0.removeFromSuperview() }
+        mainDescriptionStackView.arrangedSubviews.forEach { $0.removeFromSuperview() }
+
+        //LEFT
+        if let leftImageKey = content.getLeftImage() {
+            imageProvider.getImage(key: leftImageKey) { image in
+                self.leftImageImageView.image = image
+            }
+        }
+        if let logoImageImageViewKey = content.getLeftImageAccessory() {
+            imageProvider.getImage(key: logoImageImageViewKey) { image in
+                self.leftImageAccessoryImageView.image = image
+            }
+        }
+        
+        //MAIN
+        mainTitleLabel.text = content.getMainTitle()
+        mainSubtitleLabel.text = content.getMainSubtitle()
+        
+        if let mainDescription = content.getMainDescription(), mainDescription.count > 0 {
+            for item in mainDescription {
+                let itemContent = item.getContent()
+                let itemColor = item.getColor()?.hexaToUIColor()
+                switch item.getType().lowercased() {
+                case "image":
+                    let imageView = UIImageView()
```

Comment:
Podemos sacar cada uno de estos cases a un método? Así queda un poco más chico este. Onda algo asi: `func getImage(for content: String) -> UIView` y `func getText(for content: String) -> UIView`

Reply:
Si, totalmente de acuerdo, de hecho creo que el método quedo enorme.